src/pages/resources/migration/android/migrate-to-3x.md

```diff
@@ -100,6 +100,23 @@ Some of the APIs available in previous major versions of the Mobile SDK for Andr
 
 The `registerExtension` API for each extension that was deprecated in the 2.x version of the mobile SDK has been removed in the 3.x version of the mobile SDK. See the [Update SDK initialization](#update-sdk-initialization) section for more details.
 
+#### MobileCore
+
+| Removed API | Alternative API |
+| :------------- | :-------------- |
+|MobileCore.registerExtension| [MobileCore.registerExtensions](../../../home/base/mobile-core/api-reference.md#registerextensions) |
+| MobileCore.start| [MobileCore.registerExtensions](../../../home/base/mobile-core/api-reference.md#registerextensions) registers extensions and starts event processing by default |
+| MobileCore.dispatchEvent(Event, ExtensionErrorCallback)| [MobileCore.dispatch(Event)](../../../home/base/mobile-core/api-reference.md#registerextensions#dispatch--dispatchevent) , [MobileCore.dispatchEventWithResponseCallback(Event, long, AdobeCallbackWithError)](../../../home/base/mobile-core/api-reference.md#registerextensions#dispatch--dispatcheventwithresponsecallback) |
+| MobileCore.dispatchEvent(Event, AdobeCallback, ExtensionErrorCallback)| [MobileCore.dispatchEventWithResponseCallback(Event, long, AdobeCallbackWithError)](../../../home/base/mobile-core/api-reference.md#registerextensions#dispatch--dispatcheventwithresponsecallback) |
+| MobileCore.dispatchResponseEvent | Use **Event.Builder.inResponseToEvent(Event)** to create a response event |
+| MobileCore.dispatchEventWithResponseCallback(Event, AdobeCallbackWithError)| Use [MobileCore.dispatchEventWithResponseCallback(Event, long, AdobeCallbackWithError)](../../../home/base/mobile-core/api-reference.md#registerextensions#dispatch--dispatcheventwithresponsecallback) to explicitly specify a timeout |
+| MobileCore.log | Use logging methods exposed via **com.adobe.marketing.mobile.services.Log** |
```

Comment:
for these other APIs will the MobileCore API reference or any other samples also be updated with this PR?

Reply:
I did not intend to add any samples with this PR because the `dispatch*` api's are already linked to the docs that have example usage, and Messaging, Campaign docs will include `PresentationDelegate`, `URIHandler` related migration details.
Not sure if anyone really uses `Log`  from our SDK for us to provide an example - we can do it if needed.

Reply:
Could you include "InvalidInitException" since it is mentioned in the older launch installation instructions?

Could you link to the [Log](https://github.com/adobe/aepsdk-core-android/blob/v3.0.0-core/code/core/src/phone/java/com/adobe/marketing/mobile/services/Log.java) class for replacing logs? Also, please move it to the extension development section.

We haven't documented any APIs related to extension development in our documentation yet, but it feels incomplete without mentioning them. Could you add a subsection under MobileCore called "Extension Development" and include following content?

Core 3.0.0 is not binary compatible with extensions built using earlier versions. Third-party developers need to recompile their extensions with this version of Core.
There are certain APIs we can list, and for their replacements, we can point to our GitHub repository.
- LaunchRulesEngine
- Event.copy()
- MobileCore.log
- Mention these classes Extension, ExtensionAPI, ExtensionUnexpectedError, ExtensionListener - https://github.com/adobe/aepsdk-core-android/blob/v3.0.0-core/Documentation/EventHub/BuildingExtensions.md#building-extensions

Also add an FAQ section addressing the errors encountered when using older extensions with Core 3.0.0 and suggest solutions to resolve them?
For Adobe extensions, recommend using BOM.
For third-party extensions, advise contacting the third-party developer and recompiling with Core 3.0.0.

Reply:
Sorry for the confusion, for my initial comment I was referring to updating the API reference pages for the APIs that have been removed and include API syntax and sample for the new APIs if not already documented. For example MobileCore.log - mark this API as removed, and we should document the Log APIs, MobileCore.start, mark UserProfile APIs as removed
- https://developer.adobe.com/client-sdks/home/base/mobile-core/api-reference/
- https://developer.adobe.com/client-sdks/home/base/profile/api-reference/

Reply:
@emdobrin  - Gotcha. Praveen is going to add that in a separate PR.